server/scripts/examples.js

```diff
@@ -236,125 +234,123 @@ function createProjectsInP5user(projectsInAllCategories) {
+              },
+              {
+                name: 'sketch.js',
+                content: '// Instance Mode: Instance Container, please check its index.html file',
+                id: a,
+                _id: a,
+                fileType: 'file',
+                children: []
+              },
+              {
+                name: 'index.html',
+                content: project.sketchContent,
+                isSelectedFile: true,
+                id: b,
+                _id: b,
+                fileType: 'file',
+                children: []
+              },
+              {
+                name: 'style.css',
+                content: defaultCSS,
+                id: c,
+                _id: c,
+                fileType: 'file',
+                children: []
+              }
+            ],
+            _id: shortid.generate()
+          });
+        } else {
+          newProject = new Project({
+            name: project.projectName,
+            user: user._id,
+            files: [
+              {
+                name: 'root',
+                id: r,
+                _id: r,
+                children: [a, b, c],
+                fileType: 'folder'
+              },
+              {
+                name: 'sketch.js',
+                content: project.sketchContent,
+                id: a,
+                _id: a,
+                isSelectedFile: true,
+                fileType: 'file',
+                children: []
+              },
+              {
+                name: 'index.html',
+                content: defaultHTML,
+                id: b,
+                _id: b,
+                fileType: 'file',
+                children: []
+              },
+              {
+                name: 'style.css',
+                content: defaultCSS,
+                id: c,
+                _id: c,
+                fileType: 'file',
+                children: []
+              }
+            ],
+            _id: shortid.generate()
+          });
+        }
 
-          const assetsInProject = project.sketchContent.match(/assets\/[\w-]+\.[\w]*/g)
-            || project.sketchContent.match(/asset\/[\w-]*/g) || [];
+        const assetsInProject = project.sketchContent.match(/assets\/[\w-]+\.[\w]*/g)
+          || project.sketchContent.match(/asset\/[\w-]*/g) || [];
 
+        try {
           await addAssetsToProject(assetsInProject, res, newProject);
+        } catch (error) {
+          throw error;
+        }
 
-          newProject.save((saveErr, savedProject) => {
-            if (saveErr) throw saveErr;
-            console.log(`Created a new project in p5 user: ${savedProject.name}`);
-            projectCallback();
-          });
-        }, (categoryErr) => {
-          categoryCallback();
+        newProject.save((saveErr, savedProject) => {
+          if (saveErr) throw saveErr;
+          console.log(`Created a new project in p5 user: ${savedProject.name}`);
         });
-      }, (examplesErr) => {
-        process.exit();
```

Comment:
You removed this `process.exit`, so now when you run the `fetch-examples` script, it hangs when it's done and never closes. Could you add this back?

Reply:
I've added it back. Please let me know if there is anything else that needs to be changed.